libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt

```diff
@@ -53,6 +53,7 @@ class RouteProgress internal constructor(
     val remainingWaypoints: Int,
     val upcomingRoadObjects: List<UpcomingRoadObject>,
     val stale: Boolean,
+    internal val routeAlternativeId: String?,
```

Comment:
users may want to use this id to implement similar feature in a custom reroute controller

Reply:
the SDK hasn't exposed routeId in NavigationRoute. These 2 properties should be exposed at the same time. Let's make it after a few round of tests